modules/infrastructure-moving-router.adoc

```diff
@@ -55,14 +55,38 @@ spec:
 status:
   labelSelector: app=router,router=router-default
   replicas: 1
+
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  creationTimestamp: 2019-04-18T12:35:39Z
+  finalizers:
+  - ingresscontroller.operator.openshift.io/finalizer-ingresscontroller
+  generation: 1
+  name: default
+  namespace: openshift-ingress-operator
+  resourceVersion: "11341"
+  selfLink: /apis/operator.openshift.io/v1/namespaces/openshift-ingress-operator/ingresscontrollers/default
+  uid: 79509e05-61d6-11e9-bc55-02ce4781844a
+spec: {}
+status:
+  availableReplicas: 2
+  conditions:
+  - lastTransitionTime: 2019-04-18T12:36:15Z
+    status: "True"
+    type: Available
+  domain: apps.<cluster>.example.com
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+  selector: ingresscontroller.operator.openshift.io/deployment-ingresscontroller=default
 ----
 <1> Note that the `nodeSelector` is configured to match the `worker` label.
 
-. Edit the `clusteringress` resource and change the `nodeSelector` to use the
+. Edit the `ingresscontroller` resource and change the `nodeSelector` to use the
 `infra` label:
 +
 ----
-$ oc edit clusteringress default -n openshift-ingress-operator -o yaml
+$ oc edit ingresscontroller default -n openshift-ingress-operator -o yaml
 ----
 +
 Update the `nodeSelector` stanza to reference the `infra` label as shown:
```

Comment:
"Update" is incorrect because the default `IngressController` resource does not have any `nodeSelector` stanza.  "Add" would be better, and better still would be to indicate further that the `nodeSelector` stanza needs to be added under the `spec` stanza:
```yaml
spec:
  nodeSelector:
    matchLabels:
      node-role.kubernetes.io/infra: ""
```

Reply:
Sorry, the above yaml is incorrect—`nodeSelector` needs to be under `nodePlacement`.  The correct yaml is as follows:
```yaml
spec:
  nodePlacement:
    nodeSelector:
      matchLabels:
        node-role.kubernetes.io/infra: ""
```